CMakeLists.txt

```diff
@@ -454,10 +463,12 @@ if(NOT LIBUNWIND_FOUND)
 	message(WARNING "libunwind not found. Stack traces from tests will not be reliable")
 endif()
 
-if(TEST_DIR)
-	enable_testing()
-else()
-	message(WARNING "TEST_DIR is empty - 'make test' will not work")
+if(BUILD_LIBPMEMFILE_POSIX_TESTS OR ACTUALLY_BUILD_LIBPMEMFILE_TESTS)
```

Comment:
Please make a variable for that (TESTS_ENABLED?).

Reply:
Done